crates/iceberg/src/spec/manifest.rs

```diff
@@ -1189,6 +1211,47 @@ impl DataFile {
         self.sort_order_id
     }
 }
+
+/// Convert data files to avro bytes.
```

Comment:
I suggest to use API:

- `format_data_files_into_avro_bytes`
- `parse_data_files_from_avro_bytes`